README.md

```diff
@@ -73,7 +73,11 @@ Request Options:
       --multiplexed-thrift       Enables the Thrift TMultiplexedProtocol used
                                  by services that host multiple Thrift services
                                  on a single endpoint.
-      --stream-interval=         Interval between consecutive stream message sends, applicable separately to every stream request opened on a connection.
+      --stream-interval=         Interval between consecutive stream message sends,
+                                 applicable separately to every stream request
+                                 opened on a connection.
+      --delay-close-send-stream= Delay the closure of send stream once all the
```

Comment:
Can we call this `--stream-delay-close-send` ? I would prefer to keep having the prefix --stream for all options regarding streaming.

Reply:
makes sense, done.